WebViewJavascriptBridge/WebViewJavascriptBridge.m

```diff
@@ -345,6 +345,7 @@ - (void)webViewDidFinishLoad:(UIWebView *)webView {
     if (_numRequestsLoading == 0 && ![[webView stringByEvaluatingJavaScriptFromString:@"typeof WebViewJavascriptBridge == 'object'"] isEqualToString:@"true"]) {
         NSBundle *bundle = _resourceBundle ? _resourceBundle : [NSBundle mainBundle];
         NSString *filePath = [bundle pathForResource:@"WebViewJavascriptBridge.js" ofType:@"txt"];
+        NSAssert(filePath != nil, @"WebViewJavascriptBridge.js.txt is missing. Your bundle is probably not setup correctly (using bundle %@)", bundle);
```

Comment:
I would remove this check. In an app that I'm writing, we've pulled the js file from our application bundle and are loading it on our server. This would result in runtime errors in my application while debugging/developing locally.